Added responsiveness to the whole website

ammrit2312 opened the pull request.

ammrit2312 committed:
Initial work

RIT3shSapata commented on src/App.css:
```diff
@@ -0,0 +1,12 @@
+@import url(/Users/amritesh/Downloads/Coding/react/react-trainee/src/index.css);
```
Hey @ammrit2312 don't use absolute paths of your machine, any images or svgs that you want to use, create a copy of them in the assets folder and use relative paths of the directory.
Don't go out of the scope of the directory.

RIT3shSapata replied:
This is to prevent any issues when we change from one device to another and also helps in the deployment

RIT3shSapata reviewed (changes requested):
@ammrit2312 first up great progress, and I love the fact that you use BEM naming convention to name your classes.
And as mentioned in one of the files, use relative paths for importing images and fonts. If you download new font or image, make a copy of them in the assets folder and use the path relative to the directory, and don't use absolute paths of your machine. 
This will prevent errors when someone tried to run your code on their machine.

ammrit2312 committed:
Made changes to css and added our team

ammrit2312 committed:
added reach us

ammrit2312 commented:
So I made these changes and also reach us and I am working on other topics like footer and finally making it as responsive as possible. 
Right now I have added Reach Us and some of the places added media queries in CSS. I also added meet the team part. Will be working on the footer now and the app display section.

RIT3shSapata reviewed (commented):
Good Progress on the site 👍🏽

ammrit2312 committed:
added footer

RIT3shSapata reviewed (commented):
Hey @ammrit2312 the site looks really good on desktop. 
Just one small feedback, can you center the reach us component? The form on the left hand side is bit too much to the left side.

ammrit2312 committed:
General responsiveness

ammrit2312 committed:
Added show me changed some problems with responsive part

ammrit2312 committed:
Main Top pyramid part Done

ammrit2312 committed:
Updated file

RIT3shSapata commented:
Hey @ammrit2312 great job on the website, it looks really good. Start working on the responsiveness and the color-changing part using react context.

ammrit2312 committed:
Context API added

ammrit2312 committed:
Context API added

RIT3shSapata reviewed (commented):
Great job on implementing the context api

ammrit2312 committed:
added responsivness

ammrit2312 commented:
Hey @RIT3shSapata I have added the responsiveness to the website and made required changes. Please inform me what else is needed to be done. If any other work is left in the website then please do inform me.
